lib/iris/etc/pp_save_rules.txt

```diff
@@ -150,6 +150,31 @@ IF
 THEN
     pp.lbtim.ia = 0
 
+IF
+    # If the cell methods contain a minimum then overwrite lbtim.ia with this
+    # interval
+    scalar_coord(cm, 'time') is not None
+    scalar_coord(cm, 'time').has_bounds()
+    scalar_coord(cm, 'clim_season') is None
+    scalar_coord(cm, 'forecast_period') is not None or scalar_coord(cm, 'forecast_reference_time') is not None
+    scalar_cell_method(cm, 'minimum', 'time') is not None
+    scalar_cell_method(cm, 'minimum', 'time').intervals != ()
+    scalar_cell_method(cm, 'minimum', 'time').intervals[0].endswith('hour')
+THEN
+    pp.lbtim.ia = int(scalar_cell_method(cm, 'minimum', 'time').intervals[0][:-5])
```

Comment:
Sorry, I'm just getting to grips with Github, do you mean a comment on line 150, or comment on the new lines 153 to 164?